### PR TITLE
StockTicker

### PR DESCRIPTION
**Pull request by hollowmatt**
A simple stock ticker app.  Add three stock symbols, and an API key for Alpha vantages - and you have a stock ticker.  5 min cache for stock data.

### CODE REVIEW

**Comment by hollowmatt** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by AmillionAirs** [comment]
got a sample image of what it looks like?

File: apps/stockticker/stock_ticker.star [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                full_url = STOCK_QUOTE_URL + a + ALPHA_KEY
+                rep = http.get(full_url)
+                if rep.status_code != 200:
+                    fail("API request failed with status %d", rep.status_code)
+                rate = rep.json()["Global Quote"]["05. price"]
+                msg = msg + a + ": $" + str(rate[:-2]) + " ... "
+            cache.set("sym_rate", msg, ttl_seconds = 240)
+    else:
+        # output error
+        msg = "%s API key is required"
```
Comment (matslina) [inline apps/stockticker/stock_ticker.star:54]: Missing string interpolation?

File: apps/stockticker/stockticker.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+var source []byte
+
+// New creates a new instance of the Stock Ticker applet.
+func New() manifest.Manifest {
+	return manifest.Manifest{
+		ID:          "stock-ticker",
+		Name:        "Stock Ticker",
+		Author:      "Matt Holloway",
+		Summary:     "3 stocks scrolling",
+		Desc:        "This is a simple stock ticker app, that will display a stock ticker for 3 stock symbols.  If you want more, spin up a second copy of the app to have more stocks tick.",
```
Comment (matslina) [inline apps/stockticker/stockticker.go:20]:
```suggestion
		Desc:        "This is a simple stock ticker app, that will display a stock ticker for 3 stock symbols.  If you want more, spin up a second copy of the app to have more stocks tick. Requires a (fee) API key from alphavantage.co.",
```

**Comment by matslina** [comment]
iTerm2 ISdARw derp

Seems to work fine.

**Review by betterengineering — APPROVED** [review]
LGTM

**Pull request merged**